packages/@pollyjs/adapter-fetch/tests/integration/server-test.js

```diff
@@ -201,6 +201,42 @@ describe('Integration | Server', function() {
         /Invalid filter callback provided/
       );
     });
+
+    it('.times()', async function() {
```

Comment:
Might be worth adding a test for the precedence between `times()` + `{ times }` when both are used

Reply:
Done.